src/fu-remote-list.c

```diff
@@ -236,6 +237,17 @@ fu_remote_list_add_remote(FuRemoteList *self, FwupdRemote *remote)
 	g_ptr_array_add(self->array, g_object_ref(remote));
 }
 
+static gboolean
+fu_remote_list_is_remote_origin_lvfs(FwupdRemote *remote)
+{
+	if (g_str_has_prefix(fwupd_remote_get_id(remote), "lvfs"))
+		return TRUE;
+	if (fwupd_remote_get_metadata_uri(remote) != NULL &&
+	    g_strstr_len(fwupd_remote_get_metadata_uri(remote), -1, "fwupd.org") != NULL)
```

Comment:
I think this should be outside the LVFS in name check. What if someone renamed a remote? We can still catch it if they are using fwupd.org.